content/blog/2020-09-06-haiku_monthly_activity_report_august_2020.md

```diff
@@ -0,0 +1,106 @@
++++
+type = "blog"
+title = "Haiku Activity Report - August 2020"
+author = "bitigchi"
+date = "2020-09-06 13:00:00+03:00"
+tags = ["haiku", "monthly activity report"]
++++
+
+Welcome to the Haiku Monthly Activity Report for August 2020. This report covers revisions from hrev 54480 to hrev54538.
+
+This month is not active as the last one, but we have exciting improvements in the pipeline, since GSOC 2020 is coming to an end, and our developers are working hard to review GSOC projects at a fast pace.
+
+## Applications
+
+syedsouban, PulkoMandy, and bitigchi worked on adding search engine shortcuts to WebPositive. Now it is possible to use different search engines like DuckDuckGo, Ecosia, Bing etc.
+
+A list of supported search engines and the respective shortcuts can be found below:
+
+- Google (g)
+- Bing (b)
+- Wikipedia (w)
+- DuckDuckGo (d)
+- Baidu (a)
+- Yandex (y)
+- Ecosia (e)
+- Qwant (q)
+
+Example: To search on Ecosia, use: e <search_term>.
+
+bitigchi has committed a fix that allows rebooting immediately when clicked "Reboot" in SoftwareUpdater. Previously it displayed the regular system shutdown prompt, which was not ideal, UX-wise.
+
+Humdinger also improved accessibility on WebPositive by adding a missing tooltip on Settings window.
+
+PulkoMandy and nielx worked on making the Installer more reliable by refining CopyEngine and post-install scripts.
+
+PulkoMandy worked on fixing Pulse issues, now it is possible to resize the window without artifacts and CPU image rescale issues. PulkoMandy also improved battery detection for PowerStatus.
+
+PulkoMandy also contributed some nice interface fixes to Icon-o-Matic, it uses more native Haiku widgets, and supports longer filenames.
+
+## arm64
+
+kallisti5 continued working on making Haiku working under arm architecture. Currently Haiku is crashing early in the boot process, and it is being investigated.
+
+## Deskbar
+
+John Scipione has re-implemented the old vertical Deskbar mini mode. This version was deprecated in favor of the new horizontal mini mode, which uses only a tab-height of space. With this addition, it is now possible to use Deskbar in 4 different modes:
+
+- Default mode
+- Horizontal mini mode
+- Vertical mini mode
+- Expanded mode
+
+## Drivers
+
+Michael Lotz continued working on audio drivers. Specifically:
+
+- ac94/es1370 audio driver
+- ac94/auich audio driver
+- multi_audio buffer keeping has been improved
+
+## ffmpeg
+
+PulkoMandy has improved ffmpeg support in Haiku, fixing multiple memory leaks, asserts, and deleting unnecessary code.
+
+## GSOC 2020
+
+Following students continue to work on their respective areas:
```

Comment:
Actually this month is the end of GSoC. There will be a separate blog post (next week) summarizing the results once the final evaluations are published by Google.